opensrp-chw-core/src/main/java/org/smartregister/chw/core/presenter/NavigationPresenter.java

```diff
@@ -54,6 +54,7 @@ private void initialize() {
         tableMap.put(CoreConstants.DrawerMenu.MALARIA, CoreConstants.TABLE_NAME.MALARIA_CONFIRMATION);
         tableMap.put(CoreConstants.DrawerMenu.FAMILY_PLANNING, FamilyPlanningConstants.DBConstants.FAMILY_PLANNING_TABLE);
         tableMap.put(CoreConstants.DrawerMenu.ALL_CLIENTS, CoreConstants.TABLE_NAME.FAMILY_MEMBER);
+        tableMap.put(CoreConstants.DrawerMenu.UPDATES, CoreConstants.TABLE_NAME.CLOSE_REFERRAL);
```

Comment:
Does this mean counts/updates will **only** be from those referrals that are closed?

Reply:
Not really so by convention, the navigation menu accepts a default table to query from if the **query statement** is not overridden in the NavigationInteractor. Check the **getCount** ,method o of [NavigationInteractor](https://github.com/OpenSRP/opensrp-client-chw-core/blob/implement-referral-notification-register/opensrp-chw-core/src/main/java/org/smartregister/chw/core/interactor/NavigationInteractor.java)